recipe/meta.yaml

```diff
@@ -1,56 +1,55 @@
 {% set name = "spacy-transformers" %}
 {% set module_name = "spacy_transformers" %}
-{% set version = "1.2.4" %}
+{% set version = "1.3.5" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 859c2093fad9ff41005bc55c0018d4290bdd62454f76bddbcf318aae85f307b5
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: accdfe44a26517714c6990ec6bae88796eb348286fea7ba20ba2736f70c83fa1
+
 
 build:
   number: 0
-  # spacy isn't available on s390x
-  skip: True  # [py<36 or s390x]
-  # At present, there are no 3.11 compatible Pytorch version for win or ppc
-  # There is a bug requiring PyTorch 1.10 on osx-64 which is not available for 3.11
-  # As transformers 4.29.2 b0 has the same issue, we apply its skipping logic here  
-  skip: True  # [py>310 and (ppc64le or win or (osx and x86_64))]
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  # spacy isn't available on s390x.
+  # transformers isn't available for py>=312 currently.
```

Comment:
What does it take to build transformers for 3.12, have you tried?

Reply:
Rebuilding transformers https://github.com/AnacondaRecipes/transformers-feedstock/pull/11

Reply:
Done